.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Test Coveralls
+
```

Comment:
we can discuss this file tomorrow. Want to make sure all the steps are understood